presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -654,6 +654,83 @@ This query will collect statistics for 2 partitions with keys:
 * ``partition1_value1, partition1_value2``
 * ``partition2_value1, partition2_value2``
 
+Quick Stats
+--------------------------------------
+
+The Hive connector can build basic statistics for partitions with missing statistics
+by examining file or table metadata. For example, parquet footers can be used to infer
+row counts, number of null and min/max values. These statistics help in query planning,
+and serve as interim cache of stats for partitions which haven't yet been ANALYZEd
+
+The following properties can be used to control how these quick stats are built:
+
+.. list-table::
+   :widths: 20 70 10
+   :header-rows: 1
+
+   -
+
+      - Property Name
+      - Description
+      - Default
+   -
+
+      - ``hive.quick-stats.enabled``
+      - Enable stats collection through quick stats providers. Also
+         toggleable through the ``quick_stats_enabled`` session property
+      - ``false``
+   -
+
+      - ``hive.quick-stats.max-concurrent-calls``
+      - Quick stats are built for multiple partitions concurrently. This
+         property sets the maximum number of concurrent builds that can
+         be made
+      - ``100``
+   -
+
+      - ``hive.quick-stats.inline-build-timeout``
+      - Duration the query that initiates a quick stats build for a
+         partition should wait before timing out and returning empty
+         stats. Also toggleable through the
+         ``quick_stats_inline_build_timeout`` session property
+      - ``60s``
+   -
+
+      - ``hive.quick-stats.background-build-timeout``
+      - If a query observes that quick stats are already being built for
+         a partition, this is the duration it will wait for those stats
+         to be built before returning empty stats
+      - ``0s``
+   -
+
+      - ``hive.quick-stats.cache-expiry``
+      - Duration to retain the stats in the quick stats in-memory cache
```

Comment:
```suggestion
      - Duration to retain the stats in the quick stats in-memory cache.
```
Nit: If one item in a list needs punctuation (a period), they all do. (I wanted to explain why I made this and some of the similar nit suggestions.)